parsing/location.ml

```diff
@@ -130,78 +132,108 @@ let highlight_terminfo ppf lb locs =
   Terminfo.resume stdout !num_loc_lines;
   flush stdout
 
-(* Highlight the location by printing it again. *)
+(* Highlight the location by printing it again.
 
-let highlight_dumb ~print_chars ppf lb loc =
-  (* Char 0 is at offset -lb.lex_abs_pos in lb.lex_buffer. *)
-  let pos0 = -lb.lex_abs_pos in
-  (* Do nothing if the buffer does not contain the whole phrase. *)
-  if pos0 < 0 then raise Exit;
-  let end_pos = lb.lex_buffer_len - pos0 - 1 in
-  (* Determine line numbers for the start and end points *)
-  let line_start = ref 0 and line_end = ref 0 in
-  for pos = 0 to end_pos do
-    if Bytes.get lb.lex_buffer (pos + pos0) = '\n' then begin
-      if loc.loc_start.pos_cnum > pos then incr line_start;
-      if loc.loc_end.pos_cnum   > pos then incr line_end;
-    end
-  done;
-  Format.fprintf ppf "@[<v>";
-  (* Print character location (useful for Emacs) *)
-  if print_chars then begin
-    Format.fprintf ppf "Characters %i-%i:@,"
-                   loc.loc_start.pos_cnum loc.loc_end.pos_cnum
-  end;
-  (* Print the input, underlining the location *)
-  Format.pp_print_string ppf "  ";
-  let line = ref 0 in
-  let pos_at_bol = ref 0 in
-  for pos = 0 to end_pos do
-    match Bytes.get lb.lex_buffer (pos + pos0) with
-    | '\n' ->
-      if !line = !line_start && !line = !line_end then begin
-        (* loc is on one line: underline location *)
-        Format.fprintf ppf "@,  ";
```

Comment:
This underline-location start format is `"@,  "`, but your diff replaces it with `"@,"` only. Why?